Assets/_BForBoss/_Weapons/Scripts/Weapons/WeaponBehaviour.FiringRayCast.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
Are these directories needed?

Reply:
No, i'll be removing these in the part 2 of the PR.